scripts/all-tests.sh

```diff
@@ -80,6 +80,7 @@ case "$TEST_SUITE" in
         # Run quality task. Pass in the 'fail-under' percentage to diff-quality
         paver run_quality -p 100
 
+        paver run_complexity > code_complexity.log || echo "Unable to calculate code complexity. Ignoring error."
```

Comment:
I did glance at that, but didn't consider the plugin. Basically in terms of output, we can get an overall score, or we can get a full report for everything. I had looked at ways of capturing complexity on your diff (can't use diff-quality for it because it considers the whole method/class/etc). But ultimately I thought a straight text report, which includes a total score at the end, would be easiest, esp when thinking of an eventual move to Travis.